Description: Avoid circular import loop.
Author: Brian May <bam@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: other, https://github.com/VPAC/django-xmlrpc/commit/365c58f0791274055e3623d68d4132fd30e6e0c8
Forwarded: no
Reviewed-By: Brian May <bam@debian.org>
Last-Update: 2012-04-04

--- django-xmlrpc-0.1.orig/django_xmlrpc/__init__.py
+++ django-xmlrpc-0.1/django_xmlrpc/__init__.py
@@ -37,4 +37,4 @@ LIABILITY, WHETHER IN CONTRACT, STRICT L
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
-from views import xmlrpcdispatcher
+#from views import xmlrpcdispatcher
